sparse/_compressed/compressed.py

```diff
@@ -89,11 +89,21 @@ class GCXS(SparseArray, NDArrayOperatorsMixin):
 
     This is stored in GCXS format, a generalization of the GCRS/GCCS formats
     from 'Efficient storage scheme for n-dimensional sparse array: GCRS/GCCS':
-    https://ieeexplore.ieee.org/document/7237032. GCXS generalizes the csr/csc
-    sparse matrix formats. For arrays with ndim == 2, GCXS is the same csr/csc.
+    https://ieeexplore.ieee.org/document/7237032. GCXS generalizes the CRS/CCS
+    sparse matrix formats.
+
+    For arrays with ndim == 2, GCXS is the same CSR/CSC.
     For arrays with ndim >2, any combination of axes can be compressed,
     significantly reducing storage.
 
+    Let the 3 arrays be RO, CO and VL. The first element
+    of array RO is the integer 1 and later elements are the number of
```

Comment:
```suggestion
    of array RO is the integer 0 and later elements are the number of
```

We use 0-based indexing in Python.

Reply:
Ah, thanks!